MeetU/MeetU/API/FollowsController.cs

```diff
@@ -24,6 +31,42 @@ public IQueryable<Follow> GetFollows()
             return db.Follows;
         }
 
+        // GET: api/Following/5
+        [HttpGet]
+        [Route("api/Following/{userId}")]
+        public IQueryable<FollowingOrFollowedView> GetProfileByFollowingUserId(string userId)
+        {
+            return
+                from p in db.Profiles
+                join f in db.Follows
+                on p.UserId equals f.FollowedUserId
+                where f.FollowingUserId == userId
+                select new FollowingOrFollowedView
+                {
+                    UserId = p.UserId,
+                    NickName = p.NickName,
+                    Picture = p.Picture
+                };
+        }
+
+        //GET: api/FollowedBy/5
+        [HttpGet]
+        [Route("api/FollowedBy/{userId}")]
+        public IQueryable<FollowingOrFollowedView> GetProfileByFollowedUserId(string userId)
+        {
+            return
+                from p in db.Profiles
+                join f in db.Follows
+                on p.UserId equals f.FollowingUserId
+                where f.FollowedUserId == userId
+                select new FollowingOrFollowedView
+                {
+                    UserId = p.UserId,
+                    NickName = p.NickName,
```

Comment:
@rong4188 
Ye..
I' ll create a card for this issue, because the change from `NickName` to `Nickname` require modification on a lot of places across database and back end.